includes/abstracts/abstract-wc-settings-api.php

```diff
@@ -795,18 +744,17 @@ public function validate_text_field( $key ) {
 	 *
 	 * Make sure the data is escaped correctly, etc.
 	 *
-	 * @param  mixed $key
+	 * @param  string $key
 	 * @return string
 	 */
 	public function validate_price_field( $key ) {
+		$text      = $this->get_option( $key );
+		$field_key = $this->get_field_key( $key );
 
-		$text  = $this->get_option( $key );
-		$field = $this->get_field_key( $key );
+		if ( isset( $_POST[ $field_key ] ) ) {
 
-		if ( isset( $_POST[ $field ] ) ) {
-
-			if ( $_POST[ $field ] !== '' ) {
-				$text = wc_format_decimal( trim( stripslashes( $_POST[ $field ] ) ) );
+			if ( $_POST[ $field_key ] !== '' ) {
```

Comment:
Since we are already changing this line, we should switch it to a yoda condition to match the coding standards